doc/sphinx/source/users-guide/taskpar/coforall.rst

```diff
@@ -0,0 +1,41 @@
+:title: Users Guide: The **coforall** statement
+
+The **coforall** statement
+==========================
+
+The **coforall statement** can be used to create an arbitrary number
+of related, homogeneous tasks.  Syntactically, it mirrors the for-loop
+statement, but uses the ``coforall`` keyword in place of ``for``.
+
+Operationally, a coforall loop creates a distinct task per loop
+iteration, each of which executes a copy of the loop body.
+Mnemonically, the coforall loop can be thought of as a *concurrent
+forall* --- that is, a parallel loop in which each iteration is a
+concurrent task.
+
+As with the cobegin statement, the original task does not proceed
+until the child tasks corresponding to the coforall's iterations have
+completed.  And, as there, the original task waits only for its
+immediate children, not their descendents.
+
+.. TODO: better m-dash above?
```

Comment:
Yep.  I wasn't online, so couldn't figure out how to get an m-dash.  Apparently, the answer is to paste an m-dash into your emacs buffer...